README.md

```
@@ -1,63 +1,50 @@
 <div align="center">
-<br/>
-<img src="media-resources/app_icon.png" width="200" />
-
-</div>
-
-<div align="center">
+  <br/>
+  <img src="media-resources/app_icon.png" width="200" />
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)